app/views/external/landing.html.haml

```diff
@@ -20,12 +20,20 @@
 #top-menu.top-menu= render "layouts/menu"
 
 .page-box
+  -if FALSE
+    .container
+      %h4 Latest announcements
+      %li
+        <a href="https://eristest6.partners.org/nhlbi-continues-its-support-of-the-national-sleep-research-resource/" title="Upcoming NSRR webinar on Multipaper Spectogram Analysis of Sleep EEG">Upcoming NSRR webinar on Multipaper Spectogram Analysis of Sleep EEG</a> (September 2, 2020)
```

Comment:
Use ruby `link_to` method here instead of pure HTML.

```
%li
  = link_to "Upcoming NSRR webinar on Multipaper Spectogram Analysis of Sleep EEG", "https://eristest6.partners.org/nhlbi-continues-its-support-of-the-national-sleep-research-resource", title: "Upcoming NSRR webinar on Multipaper Spectogram Analysis of Sleep EEG"
  (September 2, 2020)
```

Reply:
Done